doc/source/netscript/netscripthacknetnodeapi.rst

```diff
@@ -87,21 +87,21 @@ it gradually upgrades those Node's to level 80, 16 GB RAM, and 8 cores
             const cnt = 8;
 
             while (ns.hacknet.numNodes() < cnt) {
-                res = ns.hacknet.purchaseNode();
+                var res = ns.hacknet.purchaseNode();
                 if (res != -1) ns.print("Purchased hacknet Node with index " + res);
                 await ns.sleep(1000);
             };
 
             ns.tprint("All " + cnt + " nodes purchased")
 
-            for (const i = 0; i < cnt; i++) {
+            for (var i = 0; i < cnt; i++) {
                 while (ns.hacknet.getNodeStats(i).level <= 80) {
                     var cost = ns.hacknet.getLevelUpgradeCost(i, 1);
                     while (myMoney() < cost) {
                         ns.print("Need $" + cost + " . Have $" + myMoney());
                         await ns.sleep(3000);
                     }
-                    res = ns.hacknet.upgradeLevel(i, 1);
+                    var res = ns.hacknet.upgradeLevel(i, 1);
```

Comment:
`res` is unused in the remainder of these. I would just remove the assignment. If you keep it, use const.